### PR TITLE
feat(MintTokens): Update `CommunityCollectibleView` according to new design

### LINKED ISSUE #10601: [Community Mint Tokens] Update `CommunityCollectibleView` according to new design
### Description
Here the new [design](https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=29468%3A613999&t=XeSCLgrYs8EVTc6v-1): 

Screenshot 2023-05-09 at 17 17 05

Some notes:
1. Title format changed. Now symbol is a title side text.
2. Back button text changed.
3. Description / boxes order changed.
4. Boxes changed (Here [design](https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=29437%3A599353&t=YKCU2ivDxQLLlqGT-1) for infinite values):
Current dev:
image
New design:
Screenshot 2023-05-10 at 18 00 46

5. Warning description changed.

**NOTE**: Look at `CommunityMintTokensSettingsPanel ` and `CommunityCollectibleView`

### PR DESCRIPTION
**Pull request by noeliaSD**
Closes #10601

### What does the PR do

- Title format changed. Now symbol is a title side text.
- Back button text changed.
- Description / boxes order changed.
- Boxes changed order and format.
- Warning description changed.

### Affected areas

Community Settings / Mint token

### Screenshot of functionality

https://github.com/status-im/status-desktop/assets/97019400/d1ea2002-d551-4205-9a7e-836622cb046a

### CODE REVIEW

File: ui/app/AppLayouts/Chat/views/communities/CommunityCollectibleView.qml [hunk 1]
```diff
@@ -143,44 +134,46 @@ StatusScrollView {
...
                     StatusBaseText {
                         Layout.maximumWidth: mainLayout.width - Style.current.padding
-                        Layout.alignment: previewBox.leftAlignment ? Qt.AlignLeft : Qt.AlignHCenter
+                        Layout.alignment: Qt.AlignLeft
                         text: previewBox.value
                         elide: Text.ElideRight
                         font.pixelSize: Theme.primaryTextFontSize
                         color: Theme.palette.directColor1
-                        horizontalAlignment: previewBox.leftAlignment ? Text.AlignLeft : Text.AlignHCenter
+                        horizontalAlignment: Text.AlignLeft
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/views/communities/CommunityCollectibleView.qml:142]: the `Text.AlignLeft` and `Qt.AlignLeft` are default; you might as well drop them

File: ui/app/AppLayouts/Chat/views/communities/CommunityCollectibleView.qml [hunk 2]
```diff
@@ -143,44 +134,46 @@ StatusScrollView {
...
-                value: root.transferable ? qsTr("Yes") : qsTr("No")
+                label: qsTr("Total")
+                value: root.infiniteSupply ? qsTr("∞") : LocaleUtils.numberToLocaleString(root.supplyAmount)
             }
 
             CustomPreviewBox {
-                label: qsTr("Remote self-destruct")
-                value: root.selfDestruct ? qsTr("Yes") : qsTr("No")
+                label: qsTr("Remaining")
+                value: root.infiniteSupply ? qsTr("∞") : LocaleUtils.numberToLocaleString(root.remainingTokens)
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/views/communities/CommunityCollectibleView.qml:160]: Here you're making the `INF` symbol translatable...

File: ui/app/AppLayouts/Chat/views/communities/CommunityMintedTokensView.qml [hunk 3]
```diff
@@ -34,11 +34,11 @@ StatusScrollView {
             }
 
             // TO REMOVE: Just added bc backend still doesn't have `availableTokens` property in model. Once it is added, the following 2 lines can be removed.
-            if(!availableTokens)
-                availableTokens = 0
+            if(!remainingTokens)
+                remainingTokens = 0
             if(supply === 0)
                 supply = "∞"
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/views/communities/CommunityMintedTokensView.qml:40]: ... but here it isn't

Personally I'd say don't :)

**Review by caybro — APPROVED** [review]
LGTM

Just some minor suggestions inline

**Review by alexjba — APPROVED** [review]
LGTM! 👍

**Comment by noeliaSD** [comment]
Pushed updates according to comments! :)

**Pull request merged**